src/libsync/propagateupload.cpp

```diff
@@ -163,27 +163,36 @@ void PropagateUploadFileCommon::setDeleteExisting(bool enabled)
     _deleteExisting = enabled;
 }
 
-
 void PropagateUploadFileCommon::start()
 {
+    /* Currently the File to upload has the same information as the SyncFileItem,
+     * but the idea is to make a separation between the local file and the file that's
+     * being uploaded, in our current case they are the same, but perhaps we can apply
+     * some filters in the future like 'resize' for pictures and so on.
+     *
+     * this by no means is a finished job, but a first step.
```

Comment:
Comments That explains what or why the code do something are great.
But this one comment should better be in the commit message as it does not really explain the code but rather the commit.